plugins/org.jkiss.dbeaver.ext.tidb/src/org/jkiss/dbeaver/ext/tidb/model/plan/TiDBPlanNodePlain.java

```diff
@@ -0,0 +1,125 @@
+package org.jkiss.dbeaver.ext.tidb.model.plan;
+
+import org.jkiss.dbeaver.ext.mysql.model.plan.MySQLPlanNode;
+import org.jkiss.dbeaver.ext.mysql.model.plan.MySQLPlanNodePlain;
+import org.jkiss.dbeaver.model.exec.plan.DBCPlanNode;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
+import org.jkiss.dbeaver.model.meta.Property;
+
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class TiDBPlanNodePlain extends MySQLPlanNode {
+    protected String id;
+    // Before 4.0 estRows named "count"
+    protected String estRows;
+    protected String task;
+    protected String accessObject;
+    protected String operatorInfo;
+
+    protected TiDBPlanNodePlain parent;
+    protected List<TiDBPlanNodePlain> nested;
+
+    public TiDBPlanNodePlain(List<TiDBPlanNodePlain> nodes) {
+        // Root node
+        id = "<plan>";
+        if (!nodes.isEmpty()) {
+            this.estRows = nodes.get(0).estRows;
+        }
+        this.nested = nodes;
+    }
+
+    public TiDBPlanNodePlain(TiDBPlanNodePlain parent, ResultSet dbResult) {
+        this.parent = parent;
+        this.id = JDBCUtils.safeGetString(dbResult, "id");
+        this.estRows = JDBCUtils.safeGetString(dbResult, "estRows");
+        this.task = JDBCUtils.safeGetString(dbResult, "task");
+        this.accessObject = JDBCUtils.safeGetString(dbResult, "access object");
+        this.operatorInfo = JDBCUtils.safeGetString(dbResult, "operator info");
+    }
+
+    @Override
+    public TiDBPlanNodePlain getParent() {
+        return parent;
+    }
+
+    @Override
+    public Number getNodeCost() {
+        return null;
+    }
+
+    @Override
+    public Number getNodePercent() {
+        return null;
+    }
+
+    @Override
+    public Number getNodeDuration() {
+        return null;
+    }
+
+    @Override
+    public Number getNodeRowCount() {
+        return Double.parseDouble(this.estRows);
+    }
+
+    @Override
+    public String getNodeName() {
+        return this.accessObject;
+    }
+
+    @Override
+    public String getNodeType() {
+        return this.id.trim().replaceAll("└", "").replaceAll("─", "");
+    }
+
+    @Override
+    public Collection<? extends DBCPlanNode> getNested() {
+        return this.nested;
+    }
+
+    @Property(order = 0, viewable = true)
+    public String getId() {
+        return id;
+    }
+
+    @Property(order = 1, name="estRows", viewable = true)
+    public String getESTRows() {
+        return estRows;
+    }
+
+    @Property(order = 2, name="task", viewable = true)
+    public String getTask() {
+        return task;
+    }
+
+    @Property(order = 3, name="access object", viewable = true)
```

Comment:
Please, remove `name`
And localize this property in the osgi-inf folder.
As an example you can take a look at this code:
https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.ext.mysql/OSGI-INF/l10n/bundle.properties#L263
Please use a capital letter for the first letter.